docs/reference/API/element.md

```diff
@@ -0,0 +1,295 @@
+|           |         |      |
+
+The `element` property returns the DOM element with the given ID.
+
+```html
+    from pyscript import Element
+
+    my_div = Element('my-div')
+    print(my_div.element)
+```
+
+## Element.id
+
+| Parameter | Default | Type |
+|-----------|---------|------|
+|           |         |      |
+
+Return the element's ID.
+
+```html
+
+<div id="my-div"></div>
+
+<py-script>
+    from pyscript import Element
+
+    my_div = Element('my-div')
+    print(my_div.id) # prints 'my-div'
+</py-script>
+```
+
+## Element.value
+
+| Parameter | Default | Type |
+|-----------|---------|------|
+|           |         |      |
+
+Return the element's value.
+
+```html
+<input id="my-input" value="hello world"></input>
+
+<py-script>
+    from pyscript import Element
+
+    my_input = Element('my-input')
+    print(my_input.value) # prints 'hello world'
+</py-script>
+```
+
+## Element.innerHtml
+
+| Parameter | Default | Type |
+|-----------|---------|------|
+|           |         |      |
+
+Return the element's inner HTML.
+
+```html
+<div id="my-innerHtml">
+    <b>hello world</b>
+</div>
+
+<py-script>
+    from pyscript import Element
+
+    my_innerHtml = Element('my-innerHtml')
+    print(my_innerHtml.innerHtml) # prints <b> hello world </b>
+</py-script>
+```
+
+## Element.write
+
+| Parameter   | Default | Type                        |
+|-------------|---------|-----------------------------|
+| `value`     |         |  `str` or `__mime_type__`   |
+| `append`    | False   | `bool`                      |
+
+Writes `value` to element and handles various mime types. This method also contains a `append` parameter, which defaults to `False`.
+
+```html
+<div id="foo"></div>
+
+<py-script>
+    from pyscript import Element
+    el = Element("foo")
+    el.write("Hello!")
+    el.write("World!") # will replace the previous content
+</py-script>
+```
+
+If we set `append` to `True`, it will create a child element.
+
+```html
+<div id="foo"></div>
+
+<py-script>
+    from pyscript import Element
+    el = Element("foo")
+    el.write("Hello!", append=True)
+
+    # This will create a child div with the id "foo-1"
+    el.write("World!", append=True)
+</py-script>
+```
+
+## Element.clear
+
+| Parameter | Default | Type |
+|-----------|---------|------|
+|           |         |      |
+
+Clears the element's value or content. For example, we can clear the value of an input element.
+
+```html
+<input id="foo" value="Hello!"></input>
+
+<py-script>
+    from pyscript import Element
+    el = Element("foo")
+    el.clear() # Removes value from input
+</py-script>
+```
+
+Or we can clear the content of a div element.
+
+```html
+<div id="foo">Hello!</div>
+
+<py-script>
+    from pyscript import Element
+    el = Element("foo")
+    el.clear() # Removes Hello from div content
+</py-script>
+```
+
+## Element.select
+
+Select element from `query`, it will look into the main Element if `from_content` is `True`.
```

Comment:
Again, I think we should specify that this is essentialy a wrapper on the JS function, as I said earlier.